graph/src/data/subgraph/mod.rs

```diff
@@ -238,7 +247,7 @@ impl<'de> de::Deserialize<'de> for SubgraphName {
     {
         let s: String = de::Deserialize::deserialize(deserializer)?;
         SubgraphName::new(s.clone())
-            .map_err(|()| de::Error::invalid_value(de::Unexpected::Str(&s), &"valid subgraph name"))
+            .map_err(|_| de::Error::invalid_value(de::Unexpected::Str(&s), &"valid subgraph name"))
```

Comment:
And this one

Reply:
@otaviopace  that's actually one of the places that I wasn't sure about :D 

I was reading [this documentation](https://docs.serde.rs/serde/de/trait.Error.html#method.invalid_value) and from my understanding the first value should be the value the we tried to serialize (which seems fine) but the other one is the "expected value", so showing something like "valid subgraph name, error: <some error>", didn't make much sense to me 🤔   

what do you think?